guidedmodules/stubs/q-files/stub_app/app.yaml

```diff
@@ -22,3 +22,10 @@ questions:
     title: Example Module
     type: module
     module-id: example
+input:
+  - id: my_input_id
+    name: Input Name
+    type: oscal
+    path: assets/filename.json
```

Comment:
@gregelin There is no `filename.json` in this PR.

Reply:
The use of non-existent `assets/filename.json` was intentional b/c content will be changed. I guess I could add a sample file with blank information.

Reply:
Replaced `assets/filename.json` with `components/my_component.json` and added a `my_component.json` file.